include/sway/commands.h

```diff
@@ -208,7 +208,7 @@ sway_cmd input_cmd_middle_emulation;
 sway_cmd input_cmd_natural_scroll;
 sway_cmd input_cmd_pointer_accel;
 sway_cmd input_cmd_repeat_delay;
-sway_cmd input_cmd_repeat_rate;
+sway_cmd input_cmd_repeat_step;
```

Comment:
Any reason you renamed this? Wayland uses `repeat`, not `step` (https://wayland.freedesktop.org/docs/html/apa.html#protocol-spec-wl_keyboard-event-repeat_info)

Reply:
I was unaware of that protocol, and have consistently uses 'repeat_step' to denote the spacing (in milliseconds) between subsequent binding invocations. The protocol does not apply in this instance
because the binding repeat parameters are, as far as I know, used only for bindings, not clients.

Reply:
I think `repeat` is better

Reply:
It seems I've incorrectly repurposed the advisory key repeat/rate information. Would you prefer:
* To create separate `binding_repeat_rate` and  `binding_repeat_delay` commands?
* To apply the advised repeat/rate information to the bindings?

Reply:
Do what you think is best so long as the language matches the Wayland protocol